src/Cake.AzureDevOps/Pipelines/AzureDevOpsBuildNotFoundException.cs

```diff
@@ -16,16 +16,6 @@ public AzureDevOpsBuildNotFoundException()
         {
         }
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref="AzureDevOpsBuildNotFoundException"/> class.
-        /// </summary>
-        /// <param name="repositoryId">ID of the repository where the pull request was searched.</param>
-        /// <param name="pullRequestId">ID of the pull request which could not be found.</param>
-        public AzureDevOpsBuildNotFoundException(Guid repositoryId, int pullRequestId)
-            : this("Pull request with ID " + pullRequestId + " not found in repository with GUID " + repositoryId)
-        {
-        }
-
```

Comment:
Why has this been removed?

Reply:
Because this is a copy/paste error. This constructor was never used and the message did not belong to build